app/components/navigation/sidebar/Sidebar.tsx

```diff
@@ -33,6 +33,11 @@ const navigation = [
     href: 'queries',
     icon: ChartSquareBarIcon
   },
+  {
+    name: 'Lisings',
```

Comment:
Such typo, much wow

Reply:
lol that was me, but yea that should. be fixed